src/sagemaker/local/image.py

```diff
@@ -668,3 +668,13 @@ def _ecr_login_if_needed(boto_session, image):
 
     cmd = "docker login -u AWS -p %s %s" % (token, ecr_url)
     subprocess.check_output(cmd, shell=True)
+
+    return True
+
+
+def _pull_image(image):
+    pull_image_command = ('docker pull %s' % image).strip()
+    print('docker command: {}'.format(pull_image_command))
```

Comment:
Shouldn't this be done through the logger?

Reply:
We set the log level to warning, so it ends up not showing up in jupyter :(.

https://github.com/aws/sagemaker-python-sdk/blob/cf652c285ca12fc0cc3c78c6126a1551b249df92/src/sagemaker/local/image.py#L45

Reply:
that's what log level are for. why force user to see this info-level message if they don't want to?